### PR TITLE
Records channelType in Flex Insights

### PR DESCRIPTION
**Pull request by nickhurlburt**
Now that we have unfortunately needed to abandon using separate TaskChannels for different channel types, we can no longer rely on the Communication Channel in Flex Insights to record the channel of a contact.  This PR uses Custom Conversation Attribute 6 to record `channelType` using the same words as we had in Communication Channel previously.

One note to remember:  we will need to localize some of these words ('Call', 'Web') for non-English helplines.

### CODE REVIEW

File: plugin-hrm-form/src/services/InsightsService.js [hunk 1]
```diff
@@ -50,7 +57,7 @@ function mergeAttributes(previousAttributes, { conversations }) {
 }
 
 export async function saveInsightsData(twilioTask, task) {
-  const conversations = buildConversationsObject(task);
+  const conversations = buildConversationsObject(twilioTask.attributes, task);
```
Comment (GPaoloni) [inline plugin-hrm-form/src/services/InsightsService.js:60]: Just out of curiosity, what's the difference between `task` and `twilioTask`?

**Review by GPaoloni — APPROVED** [review]
Looks good!

**Review by murilovmachado — APPROVED** [review]

File: plugin-hrm-form/src/services/InsightsService.js (hunk 1 above)
Comment (murilovmachado) [reply plugin-hrm-form/src/services/InsightsService.js:60]: `task` is our `contact`, `twilioTask` is really the task from twilio.

**Comment by nickhurlburt** [comment]
Deployed to staging

**Pull request merged**